src/components/About/About.module.css

```diff
@@ -1,3 +1,173 @@
 .title {
   text-align: center;
 }
+
+@keyframes ldio {
+  0% { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5763907)